src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -850,13 +851,13 @@
     <value>Made with 💗 by Microsoft and the PowerToys community.</value>
   </data>
   <data name="ColorPickerOnly.Text" xml:space="preserve">
-    <value>Color picker only</value>
+    <value>Open color picker only</value>
   </data>
   <data name="ColorPickerOnly_Description.Text" xml:space="preserve">
-    <value>Pick a color and copy the preferred format to clipboard</value>
+    <value>Pick a color from the screen and copy formatted value to clipboard</value>
   </data>
   <data name="EditorFirst_Description.Text" xml:space="preserve">
-    <value>Open the editor to view previously picked color and to pick new colors</value>
+    <value>Launch directly into the editor mode.</value>

```

Comment:
Do we keep the . for all checkbox contents or remove them? It's inconsistent now, I'm fine with both - not sure what W10 Settings uses for RadioButtons?